bubble-wrap.yaml

```diff
@@ -2692,7 +2695,6 @@ layers:
         # building footprints, pre-extrusion
         footprints:
             filter:
-                $zoom: { max: 18 }
```

Comment:
Strictly speaking, this will negatively impact performance at zooms above 18. However our rendering budget is pretty generous at these zoom levels because there's relatively little geometry on screen.